invokeai/backend/model_manager/load/model_loaders/ip_adapter.py

```diff
@@ -31,7 +32,7 @@ def _load_model(
         if submodel_type is not None:
             raise ValueError("There are no submodels in an IP-Adapter model.")
         model = build_ip_adapter(
-            ip_adapter_ckpt_path=str(model_path / "ip_adapter.bin"),
+            ip_adapter_ckpt_path=str(model_path),
```

Comment:
Foo. The model record isn't passed to `_load_model()`. I'll fix that. Here's a quick workaround.
```suggestion
            ip_adapter_ckpt_path=str(model_path if model_path.is_file() else model_path / "ip_adapter_bin"),
```

Reply:
This is handled further down in the `load_ip_adapter_tensors` function of `backend/ip_adapter/ip_adapter.py` .. Unless I am understanding this incorrectly.